mobile/lib/features/trade/channel_configuration.dart

```diff
@@ -148,6 +147,10 @@ class _ChannelConfiguration extends State<ChannelConfiguration> {
               Text(
                   "Please specify your preferred channel size, impacting how much you will be able to win up to.",
                   style: DefaultTextStyle.of(context).style),
+              const SizedBox(height: 10),
```

Comment:
I think we should incorporate that into the text above and reduce the wall of text in general. I guess we will need to do this anyways to save space for the toggled keyboard.